docs/oscar_references.bib

```diff
@@ -2662,3 +2662,13 @@ @Book{Zie95
   year          = {1995},
   doi           = {10.1007/978-1-4613-8431-1}
 }
+
+@article{EN62,
+  Author = {Eagon, J. and Northcott, D.G.},
+  Journal = {Proc. Royal Soc.},
+  Pages = {188-204},
+  Title = {Ideals defined by matrices, and a certain complex associated to them},
+  Volume = {269},
+  Year = {1962}
```

Comment:
```suggestion
  Year = {1962},
  doi = {10.1098/rspa.1962.0170}
```
since this has a doi, we can just add it. and this needs a bibtool run

Reply:
Yes, I was looking for this, but couldn't recall. Can you remind me how to do this bibtool run? Thx!

Reply:
You can look at the top of the bib file. I seem to remember that there was a comment stating the command.

Reply:
It's in the error message in the failing CI job.
`bibtool docs/oscar_references.bib -o docs/oscar_references.bib`